README.md

```diff
@@ -65,5 +65,5 @@ Please comment on the enhancement issue to:
 - link to relevant issues in other repos
 
 Please do not comment on the enhancement issue to:
-- discuss a detail of the design, code or docs. Use a linked-to-issue or PR
-  design for that
+- discuss a detail of the design, code or docs. Use a linked-to-issue or design PR
```

Comment:
Yeah, this sentence isn't very clear. It took me a bit to realize that enhancements are supposed to each have an issue tracking it beyond the enhancement proposal PR and doc. Maybe "PR for the enhancement proposal itself"?